app/helpers/persistent_volume_helper/textual_summary.rb

```diff
@@ -40,6 +40,11 @@ def textual_group_capacity
   #
   # Items
   #
+  def textual_pods_using_persistent_volume
+    link = url_for_only_path(:id => @record.id, :action => "show", :display => "container_groups")
+    textual_link(@record.container_groups, :as => ContainerGroup, :link => link)
+  end
```

Comment:
@martinpovolny `textual_link` didn't generate the right url since it is a complex relationship. Any other way to do this other than the above?